# GOAL
float64[3] object_position
float64[3] release_position
---
# RESULT
string result
---
# FEEDBACK
string phase